e2e/tests/ui_tests/default_judgement/1v1UnspecDJ_test.js

```diff
@@ -24,6 +24,11 @@ Scenario('DefaultJudgement @create-claim @e2e-1v1-dj @e2e-wa', async ({I, api})
   await I.login(config.judgeUserWithRegionId1);
   await I.amOnPage(config.url.manageCase + '/cases/case-details/' + caseId);
   await I.waitForText('Summary');
+  await I.amOnPage(config.url.manageCase + '/cases/case-details/' + caseId + '/trigger/EVIDENCE_UPLOAD_JUDGE/EVIDENCE_UPLOAD_JUDGECaseNoteSelection');
```

Comment:
you should wrap your code with 
  if (['preview', 'demo'].includes(config.runningEnv)) { }to make sure this will not get executed in master when dj goes live